anda/tools/arduino-app-lab-bin/gpl.txt

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 3, 29 June 2007
+
+ Copyright (C) 2007 Free Software Foundation, Inc. <https://fsf.org/>
+ Everyone is permitted to copy and distribute verbatim copies
+ of this license document, but changing it is not allowed.
+
+                            Preamble
+
+  The GNU General Public License is a free, copyleft license for
+software and other kinds of works.
+
+  The licenses for most software and other practical works are designed
+to take away your freedom to share and change the works.  By contrast,
```

Comment:
why

Reply:
owen said i had to do this

Reply:
please download this

Reply:
hate that fucking zip let's do this bullshit and a special fuck you to whoever decided not to put this code on a forge